src/flatpakBuilder.ts

```diff
@@ -0,0 +1,68 @@
+import { Command, CommandOptions } from './command'
+import { getAvailable } from './flatpakUtils'
+
+let FLATPAK_BUILDER_HOST_EXISTS: boolean | undefined
+let FLATPAK_BUILDER_SANBOXED_EXISTS: boolean | undefined
+
+/**
+ * Use host flatpak-builder and fallback to org.flatpak.Builder
+ *
+ * Throws an error when failed to find the path to flatpak-builder
+ */
+export function flatpakBuilderCommand(args: string[], options?: CommandOptions): Command {
```

Comment:
Can't this be just a different constructor for Command itself?

Reply:
That would work too, but I think it may be better separated this way

Reply:
The helper function could be put in a common file but I still believe this function should be a constructor for Command

Reply:
Made an experimental commit.

I'm not sure if I got what you meant but while it doesn't cause some issues, it creates circular dependencies between Command and the helper function